### PR TITLE
Centralized SES service

### PR DESCRIPTION
**Pull request by spartan-haopham**
## Summary
Centralized SES services
### Why
Centralization

## Types of Changes

- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 🕷 Bug fix (non-breaking change which fixes an issue)
- [ ] 👏 Performance optimization (non-breaking change which addresses a performance issue)
- [ ] 🛠 Refactor (non-breaking change which does not change existing behavior or add new functionality)
- [ ] 📗 Library update (non-breaking change that will update one or more libraries to newer versions)
- [ ] 📝 Documentation (non-breaking change that doesn't change code behavior, can skip testing)
- [ ] ✅ Test (non-breaking change related to testing)
- [ ] 🔒 Security awareness (changes that effect permission scope, security scenarios)

## Related Issues
https://www.notion.so/c0x12c/Infra-Centralized-Terraform-Module-15201fb05bf180908ad9d84dc34bd78c?pvs=4

### CODE REVIEW

File: aws/ses/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+## [0.1.0]() (2024-11-06)
```
Comment (spartan-ytruong) [inline aws/ses/CHANGELOG.md:5]: please update the date em

File: aws/ses/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+## [0.1.2]() (2024-12-04)
```
Comment (spartan-ytruong) [inline aws/ses/CHANGELOG.md:5]: https://github.com/spartan-stratos/terraform-modules/pull/28#discussion_r1868895843

File: aws/ses/examples/complete/main.tf [hunk 3]
```diff
@@ -0,0 +1,9 @@
+module "ses" {
+  source            = "../../"
```
Comment (spartan-ytruong) [inline aws/ses/examples/complete/main.tf:2]: please run `terraform fmt --recursive`

File: aws/ses/examples/complete/outputs.tf [hunk 4]
```diff
@@ -0,0 +1,3 @@
+output "domain_arn" {
+  value = module.ses.domain_identity_arn
+}
```
Comment (spartan-ytruong) [inline aws/ses/examples/complete/outputs.tf:3]: please add EOL to other files too e

File: aws/ses/main.tf [hunk 5]
```diff
@@ -0,0 +1,48 @@
+/*
+`aws_ses_domain_identity` creates an SES domain identity for the specified email domain.
+Used here to verify the ownership of the domain specified in `var.email_domain` for Amazon SES.
+https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ses_domain_identity
+*/
+resource "aws_ses_domain_identity" "current" {
```
Comment (spartan-ytruong) [inline aws/ses/main.tf:6]: you can follow up the convention in `README.md`. we can name it `this` here

**Review by spartan-ytruong — CHANGES_REQUESTED** [review]

**Review by spartan-toannhan — APPROVED** [review]

**Pull request merged**